test/local/routes/account.js

```diff
@@ -245,8 +252,56 @@ describe('/account/create', () => {
         }
       },
       push: mockPush
-    })
-    var route = getRoute(accountRoutes, '/account/create')
+    }, requireMocks)
+    const route = getRoute(accountRoutes, '/account/create')
+
+    return {
+      clientAddress,
+      emailCode,
+      keyFetchTokenId,
+      mockDB,
+      mockLog,
+      mockMailer,
+      mockMarketingCache,
+      mockMetricsContext,
+      mockRequest,
+      route,
+      sessionTokenId,
+      uid
+    }
+  }
+
+  it('should create an account', () => {
+    /* nodejs 6 :_(
+    const {
+      clientAddress,
+      emailCode,
+      keyFetchTokenId,
+      mockMarketingCache,
+      mockDB,
+      mockLog,
+      mockMailer,
+      mockMetricsContext,
+      mockRequest,
+      route,
+      sessionTokenId,
+      uid
+    } = setup()
+    */
```

Comment:
Are these comments because you believe our future selves will forget how destructuring works? It seems like a practice that could get out of hand if we started doing it for all future language features...

(sorry, thin-end-of-a-wedge arguments are lame, I know)